### ISSUE TITLE
TAB to navigate doesn't fully work on Firefox.

### ISSUE THREAD

**Issue by MebinAbraham** [ONSdigital/sdc-global-design-patterns#150]
### Expected behaviour
On Chrome: Tabbing on pages like summary on runner, works as expected, it works down the page.

### Actual behaviour
On Firefox when you use tabs ie on the summary page, it jumps straight to the submit button.

On Safari on the summary page, TABs does not work at all.
### Steps to reproduce the behaviour
Launch any eq survey and test the behaviour of tabs. It won't match Chrome's behaviour.

**Comment by armstrongb** [comment]
@MebinAbraham can you supply some open URLs for this or check that this also happens on the same components in the pattern library?

**Comment by armstrongb** [comment]
@MebinAbraham the tab issue in Firefox appears to be related to OS X system preferences being used by Firefox which restricts the way in which `tab` behaves.

In your Mac's `System Preferences > Keyboard > Shortcuts`, if you have "Full Keyboard Access" toggled to "Text boxes and lists only" instead of "All controls" then you won't tab to every link (including the skip to content link and progress nav etc). Setting Full Keyboard Access to "All controls" _fixes_ the problem. This setting did not affect other browsers such as Chrome/Safari.

**Comment by armstrongb** [comment]
Closing as this is an operating system issue not a pattern library issue.
